functorch/csrc/BatchRulesStopDecomposition.cpp

```diff
@@ -6,13 +6,98 @@
 // LICENSE file in the root directory of this source tree.
 
 #include <functorch/csrc/BatchRulesHelper.h>
+#include <ATen/FunctionalTensorWrapper.h>
 #include <ATen/Operators.h>
 #include <functorch/csrc/PlumbingHelper.h>
 #include <functorch/csrc/BatchedFallback.h>
 #include <ATen/core/dispatch/Dispatcher.h>
 
 namespace at { namespace functorch {
 
+at::Tensor sync_and_unwrap_functional_output(at::Tensor out_functional) {
```

Comment:
NB: we deleted this file but you can probably just plop the code into functorch/csrc/BatchRulesDecompositions.cpp